.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -69,9 +69,10 @@ AZURE_ARCHIVE_MANIFEST_ACCESS_TIER="Hot"
 INSTANCE_ID=$(kubectl get zenko ${ZENKO_NAME} -o jsonpath='{.status.instanceID}')
 
 KAFKA_CLEANER_INTERVAL=$(kubectl get zenko ${ZENKO_NAME} -o jsonpath='{.spec.kafkaCleaner.interval}')
+SORBETD_RESTORE_TIMEOUT=$(kubectl get zenko ${ZENKO_NAME} -o jsonpath='{.spec.sorbet.server.azure.restoreTimeout}')
```

Comment:
We could, but it will also depends on backbeat conrule interval and this will not make jobs going faster because we already make succeed when an object is transitioned
Eventually we can use it to change the interval in which we check if the object is transitioned or not..